assets/src/blocks/Covers/deprecated/coversV1.js

```diff
@@ -0,0 +1,39 @@
+export const coversV1 = {
+  attributes: {
+    title: {
+      type: 'string',
+    },
+    description: {
+      type: 'string',
+    },
+    tags: {
+      type: 'array',
+      default: []
+    },
+    posts: {
+      type: 'array',
+      default: []
+    },
+    post_types: {
+      type: 'array',
+      default: []
+    },
+    covers_view: {
+      type: 'string',
+      default: '1'
+    },
+    cover_type: {
+      type: 'string',
+    },
+  },
+  isEligible(attributes) {
+    return attributes.covers_view;
```

Comment:
Maybe we should start adding a version attribute :thinking: Could make it easier once we start having more versions. Then we can know the version from the block comment itself, without needing to check all the `isEligible` functions.

Reply:
Added! 🙂